line-colors.csv

```diff
@@ -183,3 +183,17 @@ vvs-ssb-stadtbahn,U6,,8-vvs020-u6,#ed028c,#ffffff,,rectangle
 vvs-ssb-stadtbahn,U7,,8-vvs020-u7,#0ab38d,#ffffff,,rectangle
 vvs-ssb-stadtbahn,U8,,8-vvs020-u8,#fed304,#000000,,rectangle
 vvs-ssb-stadtbahn,U9,,8-vvs020-u9,#bfb678,#000000,,rectangle
+mvv-mvg-tram,12,swm002,8-swm002-12,#903F97,#ffffff,,rectangle
+mvv-mvg-tram,16,swm002,8-swm002-16,#006CB3,#ffffff,,rectangle
+mvv-mvg-tram,17,swm002,8-swm002-17,#875A46,#ffffff,,rectangle
+mvv-mvg-tram,18,swm002,8-swm002-18,#20B14A,#ffffff,,rectangle
+mvv-mvg-tram,19,swm002,8-swm002-19,#EE1C25,#ffffff,,rectangle
+mvv-mvg-tram,20,swm002,8-swm002-20,#16C0E9,#ffffff,,rectangle
+mvv-mvg-tram,21,swm002,8-swm002-21,#16C0E9,#ffffff,,rectangle
+mvv-mvg-tram,23,swm002,8-swm002-23,#B3D235,#ffffff,,rectangle
+mvv-mvg-tram,25,swm002,8-swm002-25,#F48F99,#ffffff,,rectangle
+mvv-mvg-tram,27,swm002,8-swm002-27,#903F97,#ffffff,,rectangle
+mvv-mvg-tram,27,swm002,8-swm002-27,#FBA61C,#ffffff,,rectangle
+mvv-mvg-tram,28,swm002,8-swm002-28,#ffffff,#FBA822,#FBA822,rectangle
+mvv-mvg-tram,29,swm002,8-swm002-29,#ffffff,#EC1622,#EC1622,rectangle
+mvv-mvg-tram,E7,swm002,8-swm002-e7,#ffffff,#000000,#ffffff,rectangle
```

Comment:
Kannst du das bitte noch anpassen? 
`backgroundColor is not lowercase in row 186`